src/ContainerTools/ContainerGenerator.php

```diff
@@ -36,20 +25,6 @@ public function __construct(Configuration $configuration)
      */
     public function getContainer()
     {
-        return $this->container = $this->container ?: $this->buildContainer();
-    }
-
-    /**
-     * @return Container
-     */
-    private function buildContainer()
-    {
-        $builder = new Builder(
-            new ConfigurationLoader(new SymfonyContainerBuilder(), new DelegatingLoaderFactory(), new SymfonyFilesystem()),
-            new ContainerLoader(),
-            new Filesystem(new SymfonyFilesystem(), $this->configuration->getContainerFilePath())
-        );
-
-        return $builder->build($this->configuration);
+        return $this->pimple['container_builder']->build();
```

Comment:
It's not very clear what this object actually is `$this->pimple['container_builder']`.
Adding a `@var` declaration would help:
```php
/** @var Builder $builder */
$builder = $this->pimple['container_builder']->build();

return $builder;
```